nimble/host/services/auracast/include/services/auracast/ble_svc_auracast.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdint.h>
+#include "host/ble_gap.h"
+#include "host/ble_audio_common.h"
+#include "host/ble_audio_broadcast.h"
+
+struct ble_svc_auracast_create_params {
+    struct ble_audio_base *base;
+    struct ble_iso_big_params *big_params;
+    const char *name;
+    uint8_t adv_instance;
+};
+
+/**
+ * @brief Create Auracast Endpoint and configure advertising instance
+ *
+ * This function configures advertising instance for extended and periodic
+ * advertisements to be ready for Auracast broadcast.
+ *
+ * @param[in] params            Pointer to a `ble_svc_auracast_create_params`
+ *                              structure that defines BIG and broadcast name.
+ * @param[out] config_out       Pointer to a `ble_broadcast_base_config`
+ *                              structure to return configuration of created
+ *                              Auracast advertisement.
+ * @param[in] destroy_cb        Optional callback to be called when Auracast
+ *                              advertisement is destroyed.
```

Comment:
broadcast is still valid even without advertisements BAP 1.0 `Table 6.2: Broadcast Audio Stream states` , `Continuous transmission of the EA is not required.` and `Continuous transmission of the PA is not required.`

Reply:
That is true, BAP allows that. But for Public Broadcaster we want to make sure broadcast source is visible :) therefore PA will be there whenever auracast is created.